package/boot/rkbin/Makefile

```diff
@@ -57,19 +57,12 @@ define Trusted-Firmware-A/rk3568-e25
   TPL:=rk35/rk3568_ddr_1560MHz_uart2_m0_115200_v1.21.bin
 endef
 
-define Trusted-Firmware-A/rk3588
-  BUILD_SUBTARGET:=armv8
-  ATF:=rk35/rk3588_bl31_v1.45.elf
-  TPL:=rk35/rk3588_ddr_lp4_2112MHz_lp5_2400MHz_v1.16.bin
-endef
-
 TFA_TARGETS:= \
 	rk3308 \
 	rk3308-rock-pi-s \
 	rk3566 \
 	rk3568 \
-	rk3568-e25 \
-	rk3588
+	rk3568-e25
 
 TFA_MAKE_FLAGS+= \
```

Comment:
I guess this should be removed too, this package even does not require compile

Reply:
Not really related to this PR, but I guess it can go in a separate commit.